src/frontend/src/routes/(new-styling)/recovery-phrase/+page.svelte

```diff
@@ -287,19 +341,53 @@
           {/each}
         </div>
         <div class="flex flex-row gap-2">
-          <Button class="w-full" variant="tertiary" onclick={toggleAll}>
+          <Button
+            disabled={loading}
+            class="w-full"
+            variant="tertiary"
+            onclick={toggleAll}
+          >
             {#if showAll}
               {$t`Hide all`}
             {:else}
               {$t`Show all`}
             {/if}
           </Button>
-          <Button class="w-full" variant="tertiary" onclick={handleClearAll}>
+          <Button
+            disabled={loading}
+            class="w-full"
+            variant="tertiary"
+            onclick={handleClearAll}
+          >
             {$t`Clear all`}
           </Button>
         </div>
       </div>
-      <Button size="xl" variant="secondary">{$t`Cancel`}</Button>
+      <Button size="xl" variant="secondary" disabled={loading}
+        >{$t`Cancel`}</Button
+      >
```

Comment:
When prettier breaks components like this, you can manually put the content on it's own line. Prettier will not break it afterwards.

```suggestion
      <Button size="xl" variant="secondary" disabled={loading}>
        {$t`Cancel`}
      </Button>
```